crates/crochet/tests/pass/regex.d.ts

```diff
@@ -0,0 +1,2 @@
+export declare const match: ReadonlyRegExpMatchArray | null;
```

Comment:
This type should be `RegExpMatchArray | null` since there is no `ReadonlyRegExpMatchArray`.

Reply:
Actually, instead of prefixing the type with `Readonly`, let's wrap it in a helper type like so:
```
Readonly<RegExpMatchArray> | null
```